docs/installation/index.md

```diff
@@ -5,10 +5,20 @@ sidebar: sidebars/documentation.html
 script: osmenu.js
 ---
 
+{% if site.version == null %}
+# Katello Nightly Installation
+{% else %}
 # Katello Installation
+{% endif %}
+
+{{ site.version }}
+
+{% if site.version == null %}
+  <h3>These are the instructions for installing the unstable nightly release of katello.</h3>
```

Comment:
Maybe color this differently? Or use an alert style setup for it? The bold doesn't draw my attention like another color would.